python/example_code/neptune/tests/test_start_db_cluster.py

```diff
@@ -0,0 +1,90 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+
+import pytest
+from unittest.mock import MagicMock, patch
+from botocore.exceptions import ClientError
+
+from neptune_scenario import start_db_cluster  # Update import if needed
+
+# Speed up test runs
+POLL_INTERVAL_SECONDS = 0.1
+TIMEOUT_SECONDS = 0.3
+
+
+@patch("NeptuneScenario.time.sleep", return_value=None)  # mock sleep
+@patch("NeptuneScenario.POLL_INTERVAL_SECONDS", POLL_INTERVAL_SECONDS)
+@patch("NeptuneScenario.TIMEOUT_SECONDS", TIMEOUT_SECONDS)
+def test_start_db_cluster(mock_sleep):
+    """
+    Unit test for start_db_cluster().
+    Covers success, timeout, start failure, and paginator failure.
+    """
+    # --- Success case ---
+    mock_neptune = MagicMock()
+    paginator_mock = MagicMock()
+    mock_neptune.get_paginator.return_value = paginator_mock
```

Comment:
We want to use the established stubber pattern for the client responses, see the files in the `test_tools` directory, and add a file such as `neptune_stubber.py` for each client to stub the responses.

Reply:
This did nto work from that folder. Only way it worked was to get it into local folder. No idea why this

Reply:
Other examples won't be able to access the stubber unless it is in the correct location. Check your imports and make sure you are running the tests as instructed in the language level README.

Reply:
All stubbers in correct location.